zerver/outgoing_webhooks/generic.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from typing import Any, Dict, Text, Tuple, Callable, Mapping
+import json
+from requests import Response
+from zerver.models import UserProfile
+
+from zerver.lib.outgoing_webhook import OutgoingWebhookServiceInterface
+
+class Generic(OutgoingWebhookServiceInterface):
+
+    def __init__(self, base_url, token, user_profile, service_name):
+        # type: (Text, Text, UserProfile, Text) -> None
+        self.user_profile = user_profile # type: Text
```

Comment:
Since the superclass already has these fields, would it make sense to do the following instead?

```
super(Generic, self).__init__(base_url, token, user_profile, service_name)
```